docs/ref/contrib/gis/install/geolibs.txt

```diff
@@ -12,7 +12,7 @@ Program                   Description                           Required
 `PROJ`_                   Cartographic Projections library      Yes (PostgreSQL and SQLite only)  6.x, 5.x, 4.x
 :doc:`GDAL <../gdal>`     Geospatial Data Abstraction Library   Yes                               3.1, 3.0, 2.4, 2.3, 2.2, 2.1, 2.0
 :doc:`GeoIP <../geoip2>`  IP-based geolocation library          No                                2
-`PostGIS`__               Spatial extensions for PostgreSQL     Yes (PostgreSQL only)             3.0, 2.5, 2.4, 2.3
+`PostGIS`__               Spatial extensions for PostgreSQL     Yes (PostgreSQL only)             3.0, 2.5, 2.4
```

Comment:
Can we add PostGIS 3.1 ?

Reply:
That'll be handled in a separate PR.

Reply:
No in this PR, that is a separate issue not related with dropping support for PostGIS 2.3. Feel-free to create a ticket.

Reply:
ok thanks